charts/kube-prometheus-stack/README.md

```diff
@@ -220,6 +220,19 @@ To do so, you can set `prometheus.prometheusSpec.podMonitorSelectorNilUsesHelmVa
 
 ## Migrating from stable/prometheus-operator chart
 
+## Zero downtime
+
+Since `kube-prometheus-stack` is fully compatible with the `stable/prometheus-operator` chart a migration without downtime can be achieved.
+However the old name prefix needs to be kept. If you want the new name please follow the step by step guide bellow (with downtime).
+
+You can override the fullname to achieve this:
+
+```
```

Comment:
you need to specify your language on this line.

Reply:
Done